pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -57,6 +57,15 @@ func (a *Assignment) Borrows() bool {
 	return a.Borrowing
 }
 
+func (a *Assignment) podSetByName(psName string) *PodSetAssignment {
```

Comment:
Can we adjust naming since we return `PodSetAssignment`?

Reply:
Since this is a function on Assignment I wanted to avoid repeating "Assignment", but I'm ok with some alternative, maybe:
-  `podSetAssignment`
- `podSetAssignmentByName`
wdyt?

Reply:
I'm fine with `podSetAssignmentByName`, thanks

Reply:
done